throughput/src/bin/t_pub_thr.rs

```diff
@@ -56,18 +62,27 @@ impl TransportEventHandler for MySH {
     }
 }
 
-#[derive(Debug, StructOpt)]
-#[structopt(name = "s_pub_thr")]
+#[derive(Debug, Parser)]
+#[clap(name = "s_pub_thr")]
 struct Opt {
-    #[structopt(short = "c", long = "connect")]
```

Comment:
Actually this flag (-c) doesn't work in Zenoh rest. The common flag to used is -l.

Reply:
I've actually fixed the zenoh tests in [the forked repo under the branch_0.6.0-dev](https://github.com/william-wyliang/zenoh-tests/commit/d55228bf51a9361c428cff5c94bf949ae2a9bf70). But this can be reverted if needed. 
I think the discussion here will be: which of `locator` or `connect` is better for the new convention?

Reply:
The discussion will be closed here and moved to @jerry73204 's comments below.